packages/tracing/src/browser/browsertracing.ts

```diff
@@ -124,7 +134,8 @@ const DEFAULT_BROWSER_TRACING_OPTIONS: BrowserTracingOptions = {
   routingInstrumentation: instrumentRoutingWithDefaults,
   startTransactionOnLocationChange: true,
   startTransactionOnPageLoad: true,
-  _experiments: { enableLongTask: true, enableInteractions: false },
+  enableLongTask: true,
+  _experiments: {},
```

Comment:
m: Did we want to remove `enableInteractions`?

Reply:
That is false by default, so should be the same as not having it at all, I figured?

Reply:
yeah good point, save some bytes!